inference-engine/src/vpu/common/include/vpu/ngraph/utilities.hpp

```diff
@@ -5,11 +5,20 @@
 #pragma once
 
 #include "ngraph/node.hpp"
+#include "ngraph/type/element_type.hpp"
+
+namespace vpu {
+
+namespace utilities {
```

Comment:
Are you sure we need this namespace?

Reply:
Well, actually I don't think so, removed.